client/src/modules/templates/modals/project.modal.html

```diff
@@ -41,6 +42,46 @@
       </div>
     </div>
 
+    <!-- project logo -->
+    <div class="form-group">
+      <div style="display: flex; flex-direction: column; justify-content: center; align-items:center; margin-bottom: 10px;">
+        <div ng-if="!$ctrl.project.logo && !$ctrl.hasThumbnail" class="square100" style="display:flex; flex-direction:column; justify-content:center; align-items:center; border: 1px dotted gray; background:#efefef;">
+          <i class="fa fa-3x fa-file-image-o"></i>
+        </div>
+        <div ng-if="!$ctrl.project.logo && $ctrl.hasThumbnail">
+          <img ngf-thumbnail="{{ '$ctrl.thumbnail' }}" class="square100">
+        </div>
+        <div ng-if="$ctrl.project.logo && !$ctrl.hasThumbnail">
+          <img ng-src="{{ $ctrl.project.logo }}" class="square100" onError="this.src='assets/empty.png';" >
```

Comment:
Can you move the `this.src` into a function?  Something like `$ctrl.loadDefaultLogo()` would be nice.

It's not clear to me what `this` is set to in the callback.  I assume the DOM element, but it's hard to know when you mix AngularJS and raw JS.